smartcontractkit/external-adapters-js
# Update Synthetix Debt Pool EA to reflect SIP-165 changes

## Closes 29183

## Description

SIP-165 has been updated to require two oracles.  The first oracle will aggregate the total debt issued across multiple chains and the second calculates the debt ratio across all chains.  This PR adds support for this by creating two endpoints in the Synthetix Debt Pool EA named `debt` and `debt-ratio`.

https://sips-git-165-oracle-synthetixio.vercel.app/sips/sip-165/

## Changes

- Add two endpoints to Synthetix Debt Pool EA named `debt` and `debt-ratio`.
- Update unit and integration tests
- Refactor EA to use the SynthetixIO TypeScript library

## Steps to Test

1. Set env vars as instructed in the ReadME and run the EA
2. Verify successful response from the EA in the following scenarios
  - Get debt from a single chain
  - Get debt across multiple chains
  - Get debt ratio from a single chain
  - Get debt ratio across multiple chains

## Quality Assurance
